i18n/sl.js

```diff
@@ -0,0 +1,95 @@
+export default {
+  lang: 'sl',
+  label: {
+    clear: 'Počisti',
+    ok: 'Vredu',
+    cancel: 'Prekliči',
+    close: 'Zapri',
+    set: 'Postavi',
+    select: 'Izberi',
+    reset: 'Obnovi',
+    remove: 'Odstrani',
+    update: 'Posodobi',
+    create: 'Ustvari',
+    search: 'Išči',
+    filter: 'Filtriraj',
+    refresh: 'Ponovno naloži'
+  },
+  date: {
+    days: 'Nedelja_Ponedeljek_Torek_Sreda_Četrtek_Petek_Sobota'.split('_'),
+    daysShort: 'Ned_Pon_Tor_Sre_Čet_Pet_Sob'.split('_'),
+    months: 'Januar_Februar_Marec_April_Maj_Junij_Julij_August_September_October_November_December'.split('_'),
+    monthsShort: 'Jan_Feb_Mar_Apr_Maj_Jun_Jul_Aug_Sep_Oct_Nov_Dec'.split('_'),
+    firstDayOfWeek: 1, // 0-6, 0 - Nedelja, 1 Ponedeljek, ...
+    format24h: true
+  },
+  pullToRefresh: {
+    pull: 'Za ponovno nalaganje povleci navzdol',
+    release: 'Za ponovno nalaganje pusti',
```

Comment:
First: Thank you Morgosh for translations! One less thing on my tasklist.
Suggestion: "pusti" => "spusti"